start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir /mnt/opsbox/.aws /mnt/opsbox/.kube
```

Comment:
you've created symlinks in Dockerfile , so for what it needs? :)

Reply:
Please read README.md.
It is required for storing all credetials used in container as a separate folder n the host.